Integrations/Cybereason/Cybereason.py

```diff
@@ -1029,10 +1027,22 @@ def query_file_command():
             if 'companyName' in simple_values:
                 company_name = simple_values['companyName']['values'][0]
 
+            created_time = None
+            if 'createdTime' in simple_values:
+                created_time = timestamp_to_datestring(simple_values['createdTime']['values'][0])
```

Comment:
lets change the direct key access to `values` to `.get()` and to index `0` by checking the value of `values` is a list